app/policies/participant_policy.rb

```diff
@@ -1,11 +1,26 @@
 # frozen_string_literal: true
 
-class ParticipantPolicy < ApplicationPolicy
+class ParticipantPolicy < UserPolicy
```

Comment:
So we have ParticipantPolicy but no Participant... We'll need to revisit that later...